target/bin/dailysummery

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+test -x /usr/sbin/pflogsumm || exit 0
+test -x /usr/bin/bsd-mailx || exit 0
+
+/usr/sbin/pflogsumm /var/log/mail/mail.log -d yesterday --problems-first | /usr/bin/bsd-mailx -s "Daily Summery for HOSTNAME"  DAILY_SUMM_EMAIL
```

Comment:
Should the variable be: `$DAILY_SUMM_EMAIL` ?

Reply:
I got problems with sed when used $. Removing it fixed the error..

Reply:
Even if you encapsulate the variable with `"`?

Reply:
Used the following line and it does not replace the variable:
sed -i -r 's/$HOSTNAME/'$HOSTNAME'/g' /usr/local/bin/postfix-summary
and
sed -i -r 's/\$HOSTNAME/'$HOSTNAME'/g' /usr/local/bin/postfix-summary